tests/AutoReview/TransformerTest.php

```diff
@@ -63,4 +64,40 @@ public function provideTransformerCases()
 
         return $transformersArray;
     }
+
+    /**
+     * @requires PHP 7.1
```

Comment:
why we don't want to test if priority is set properly below 7.1 ?
like, if I want to run tool at PHP 7.0, I still want transformers to be applied in proper order, right ?

Reply:
It's not about testing if certain functionality works at a certain version, it's just a requirement for the test itself because of the way it works, priority shouldn't change with versions anyway.

However like I said, I can probably rework this.